app/claasss.py

```diff
@@ -0,0 +1,21 @@
+class Customer:
+    def __init__(self, name: str,
+                 product_cart: dict,
+                 location: list,
+                 money: int,
+                 car: dict) -> None:
+        self.name = name
+        self.product_cart = product_cart
+        self.location = location
+        self.money = money
+        self.car = car
+
+
+class Shops:
```

Comment:
The class name 'Shops' should be singular ('Shop') to follow the convention of naming classes in singular form, which represents a single entity.